rosplan_planning_system/test/hierarchical_domains/chimp/restaurant/serve_coffee_problem.pdl

```diff
@@ -0,0 +1,104 @@
+(Problem
```

Comment:
Should this file be "pdl" file extension?

Reply:
Same question applies to the other files.

Reply:
Same as  for the ddl also the problem file is in CHIMP's pdl-format.

Reply:
I suspected so - just wanted to double-check.